src/css/guide/content.css

```diff
@@ -132,7 +132,7 @@
 	}
 }
 
-@media (min-width: 45rem) {
+@media (min-width: 768px) {
```

Comment:
Is there any specific drawback I'm not aware of?

`768px` is quite a right measure considering the width of the guide's `.content`. Soon below it, it becomes too narrow. And it is also a common viewport width.